secantTests/AppTests/AppTests.swift

```diff
@@ -82,23 +45,12 @@ class AppTests: XCTestCase {
             removeItem: { _ in }
         )
 
-        let keysMissingEnvironment = AppEnvironment(
-            audioServices: .silent,
+        let walletState = AppReducer.walletInitializationState(
             databaseFiles: .live(databaseFiles: DatabaseFiles(fileManager: wfmMock)),
-            deeplinkHandler: .live,
-            derivationTool: .live(),
-            diskSpaceChecker: .mockEmptyDisk,
-            feedbackGenerator: .silent,
-            mnemonic: .mock,
-            recoveryPhraseRandomizer: .live,
-            scheduler: Self.testScheduler.eraseToAnyScheduler(),
-            SDKSynchronizer: TestWrappedSDKSynchronizer(),
             walletStorage: .throwing,
-            zcashSDKEnvironment: .testnet
+            zcashSDKEnvironment: .mainnet
```

Comment:
Why did you switch from testnet to mainnet in this test? Is that intentional?

Reply:
+1

Reply:
some copy paste issue, testnet is back